src/sidebar/components/annotation-header.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+var annotationMetadata = require('../annotation-metadata');
+var memoize = require('../util/memoize');
+var persona = require('../filter/persona');
+
+// @ngInject
+function AnnotationHeaderController(groups, settings, serviceUrl) {
+  var self = this;
+
+  this.user = function () {
+    return self.annotation.user;
+  };
+
+  this.username = function () {
+    return persona.username(self.annotation.user);
+  };
+
+  this.isThirdPartyUser = function () {
+    return persona.isThirdPartyUser(self.annotation.user, settings.authDomain);
+  };
+
+  this.serviceUrl = serviceUrl;
+
+  this.group = function () {
+    return groups.get(self.annotation.group);
```

Comment:
And I think `documentMeta` and `htmlLink` below looks like potentials for the same treatment too (which could remove the `annotationMetadata` dependency from the component)